RoddyCore/src/de/dkfz/roddy/knowledge/methods/GenericMethod.groovy

```diff
@@ -78,6 +78,15 @@ class GenericMethod {
     }
 
     /**
+     * If you need a file group output and the files are already set in the cfg, then this is the right method to call.
```

Comment:
Well where should I put them? These are convenience methods. And filegroups need to be treated different than regular calls.